static/js/recent_topics_ui.js

```diff
@@ -255,7 +256,6 @@ function format_topic(topic_data) {
     const all_senders = recent_senders.get_topic_recent_senders(stream_id, topic);
     const senders = all_senders.slice(-MAX_AVATAR);
     const senders_info = people.sender_info_for_recent_topics_row(senders);
-
     // Collect extra senders fullname for tooltip.
     const extra_sender_ids = all_senders.slice(0, -MAX_AVATAR);
     const displayed_other_senders = extra_sender_ids.slice(-MAX_EXTRA_SENDERS);
```

Comment:
Why remove black new lines? They help in reading code sometimes.

Reply:
It must have been by mistake. Resolved!!